### ISSUE TITLE
UserJS/RFP seems not to work with Multi-Account Containers, found possible workaround but not sure if can cause issues

### ISSUE THREAD

**Issue by 0brien** [arkenfox/user.js#1348]
TL;DR: does using Multi-Account Containers add-on in order to benefit from MozillaVPN integration mess up anything important in the UserJS?

Hi, I have a couple of questions about the User JS, RFP and the Multi-Account Containers (MAC) add-on. Not sure if they require MAC expertise or if there are clear answers to these questions but I figure I may as well ask.

In short, I use the MAC add-on rather than using the containers built into the FF settings so that each container can use a different VPN with MozillaVPN. However, I noticed that when RFP is enabled I can’t add MAC. It tells me ‘You need an updated version of Firefox for this extension.’

If I disable RFP, I can add MAC, and if I later re-enable RFP I can keep MAC and it seems to work with no actual issues as far as I can tell. My guess is that the reason for the error message is that RFP spoofs the user agent, giving an old version of FF. So as a result the FF add-ons store thinks MAC is not compatible with my install of FF, but in reality it is.

My questions are:

i) Is my understanding of why I am getting this error message correct or is there some known actual incompatibility between MAC and RFP/the UserJS?

ii) is there a broader risk of anything going wrong, either with MAC or with RFP or the Arkenfox UserJS more broadly, from a privacy or security POV as a result of my using them together? I normally wouldn’t think twice but the fact that the extension doesn’t show up when I have RFP enabled makes me wonder if it is something more than just the spoofed user agent that is an issue. (One example that comes to mind - I wonder if MAC might interfere with dFPI that Arkenfox enables.)

Am using up-to-date versions of both the UserJS and FF btw.

Thanks in advance!

**Comment by Thorin-Oakenpants** [comment]
duplicate of #1333 

once you have installed MAC, then it should work fine. If there are issues with dFPI then that's an upstream MAC/Mozilla bug

**Comment by Thorin-Oakenpants** [comment]
title: UserJS/RFP seems not to work with Multi-Account Container
comment: and if I later re-enable RFP I can keep MAC and it seems to work with no actual issues as far as I can tell

🤷 :)

**Comment by 0brien** [comment]
Thanks for clarifying - apologies for the duplicate post, can't believe i didn't see it
